tools/json2csv.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+from pathlib import Path
+from collections import defaultdict
+import json
+
+import numpy as np
+import pandas as pd
```

Comment:
pandas and numpy aren't requirements for eve, maybe add them to the requirements.txt so the main docker container can run the tool? Otherwise I'm not opposed to a requirements file for the tools directory since it seems it might be used outside the containers.

Reply:
Yeah, I see this more as a place to have the script, not as a dependency.